### PR TITLE
convert feedback to a static page

### PR DESCRIPTION
**Pull request by deniak**
https://validator.w3.org/feedback is a perl script but it looks like it can be a simple static page.
I took the opportunity to update the footer/header of the other pages as well.

### CODE REVIEW

File: htdocs/feedback.html [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+            </ol>
+          </div>
+
+          <h3 id="needhelp">Finding help on validation and Web authoring</h3>
+          <p>Your page doesn't validate, and you don't know why, or
+          you have a question about <abbr title="HyperText Markup Language">HTML</abbr>, 
+          stylesheets or validation?</p>
+          <p>
+          <strong>First, check our <a href="docs/help.html">Help and <abbr title="Frequently Asked Questions">FAQ</abbr> document</a>, as well as the
+            <a href="http://www.htmlhelp.com/faq/">Web authoring <abbr title="Frequently Asked Questions">FAQ</abbr></a>
```
Comment (vivienlacourba) [inline htdocs/feedback.html:27]:
```suggestion
            <a href="https://www.htmlhelp.com/faq/">Web authoring <abbr title="Frequently Asked Questions">FAQ</abbr></a>
```

File: htdocs/feedback.html [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+                            free to <a title="Send a message to the www-validator mailing list"
+                                       href="mailto:www-validator@w3.org">send it directly to the list</a>. But whatever you do, 
+                                     <strong>always use the <a href="https://www.w3.org/Search/Mail/Public/search?index-type=t;type-index=www-validator">mail
+                                         search engine</a></strong> first to check for existing messages on a given topic.
+          </p>
+
+          <p>
+          If you just want to have an informal discussion with developers and users of the Validator, 
+          you may also join the IRC channel #validator on the 
+          <a href="http://freenode.net/using_the_network.shtml">freenode</a> network (irc.freenode.net).
```
Comment (vivienlacourba) [inline htdocs/feedback.html:128]:
```suggestion
          <a href="https://freenode.net/">freenode</a> network (irc.freenode.net).
```

File: htdocs/footer.html [hunk 3]
```diff
@@ -1,55 +1,33 @@
...
-	<p id="support_logo">
-		<a href="http://www.w3.org/QA/Tools/Donate">
-			<img src="//www.w3.org/QA/Tools/I_heart_validator" alt="I heart Validator logo" title=" Validators Donation Program" width="80" height="15" />
-		</a>
-	</p>
-	<p id="version_info">
-		This service runs the W3C Markup Validator, <a href="<!--#echo var="relroot" -->whatsnew.html#v14"><abbr title="version">v</abbr>1.3+hg</a>.
-	</p>
+    <p id="activity_logos">
+    <a href="https://status.w3.org/" title="W3C's Open Source, bringing you free Web quality tools and more"><img src="https://www.w3.org/assets/logos/w3c/w3c-no-bars.svg" alt="W3C" width="72" height="47"><img src="./images/opensource-55x48.png" alt="Open-Source" title="We are building certified Open Source/Free Software. - see www.opensource.org" width="55" height="48"></a>
```
Comment (vivienlacourba) [inline htdocs/footer.html:14]: Link to the W3C Open Source software page (and not to W3C Systems Status page).

```suggestion
    <a href="https://www.w3.org/Status" title="W3C's Open Source, bringing you free Web quality tools and more"><img src="https://www.w3.org/assets/logos/w3c/w3c-no-bars.svg" alt="W3C" width="72" height="47"><img src="./images/opensource-55x48.png" alt="Open-Source" title="We are building certified Open Source/Free Software. - see www.opensource.org" width="55" height="48"></a>
```

File: htdocs/footer.html [hunk 4]
```diff
@@ -1,55 +1,33 @@
...
+    </p>
 
-	<p class="copyright">
-		<a rel="license" href="http://www.w3.org/Consortium/Legal/ipr-notice#Copyright">Copyright</a> &copy; 1994-2013
-		<a href="http://www.w3.org/"><abbr title="World Wide Web Consortium">W3C</abbr></a>&reg;
+    <p id="support_logo">
+    <a href="https://www.w3.org/donate/">
+      <img src="https://www.w3.org/QA/Tools/I_heart_validator" alt="I heart Validator logo" title=" Validators Donation Program" width="80" height="15">
```
Comment (vivienlacourba) [inline htdocs/footer.html:19]:
```suggestion
      <img src="https://www.w3.org/QA/Tools/I_heart_validator" alt="I heart Validator logo" title="Donate to W3C" width="80" height="15">
```

File: misc/testsuite/lib/__pycache__/Documentation.cpython-311.pyc
Comment (vivienlacourba) [inline misc/testsuite/lib/__pycache__/Documentation.cpython-311.pyc:1]: Does the `misc/testsuite/lib/__pycache__/` directory really needs to be committed to thsi repo?

If not it should be added to a `.gitignore` file. I also note that there is an existing `.hgignore` file that should be renamed/converted to `.gitignore`.

**Review by vivienlacourba — CHANGES_REQUESTED** [review]

File: misc/testsuite/lib/__pycache__/Documentation.cpython-311.pyc
Comment (deniak) [reply misc/testsuite/lib/__pycache__/Documentation.cpython-311.pyc:1]: Indeed. This is now addressed

Comment (vivienlacourba) [reply misc/testsuite/lib/__pycache__/Documentation.cpython-311.pyc:1]: With `*.gz` being ignored I guess you similarly need to remove from this MR the compressed scripts and css files, namely:
- `scripts/combined.js.gz`
- `style/base.css.gz`
- `style/results.css.gz`

Comment (deniak) [reply misc/testsuite/lib/__pycache__/Documentation.cpython-311.pyc:1]: There's a target in the makefile to clean those so I guess we can delete these files.
Done in https://github.com/w3c/markup-validator/pull/84/commits/b780610ae319d8aa574858a0d5a791bbfba7d2c7

**Review by vivienlacourba — APPROVED** [review]

**Pull request merged**